Tests/QueryFilter/QueryFilterTest.php

```diff
@@ -92,6 +92,49 @@ public function testGetDataSimpleBaseCase()
         self::assertSame(1000, $response->getMeta()['total_records']);
     }
 
+    /**
+     * @test
+     * @expectedException \Artprima\QueryFilterBundle\Exception\UnexpectedValueException
+     */
+    public function testGetDataInvalidFilter()
+    {
+        $queryFilter = new QueryFilter(Response::class);
+
+        $config = new BaseConfig();
+        $request = new Request(new HttpRequest([
+            'filter' => [
+                'c.unknownColumn' => 'the road to hell',
+            ],
+        ]));
+        $config->setRequest($request);
+        $config->setSearchAllowedCols(['c.dummy']);
+        $config->setStrictColumns(true);
+
+        $response = $queryFilter->getData($config);
+        $response->getData();
+    }
+
+    /**
+     * @test
+     * @expectedException \Artprima\QueryFilterBundle\Exception\UnexpectedValueException
+     */
+    public function testGetDataInvalidSortColumn()
```

Comment:
Would you please also add a test for invalid sort type?

Reply:
I wanted to, but you already did that. Look at `RequestTest.php` file to method called `testInvalidSortDirException1` and `testInvalidSortDirException2`

Reply:
I see, ok, let's keep it as it is.